README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
well, it’s 2016, unicode is 12 years old and projects are still using cp-something encodings instead of UTF-8. that’s just a wildly anachronistic environment you’re dealing with.